lunar/distribution.yaml

```diff
@@ -59,6 +59,22 @@ repositories:
       url: https://github.com/ros/genmsg.git
       version: indigo-devel
     status: maintained
+  gennodejs:
+    doc:
+      type: git
+      url: https://github.com/RethinkRobotics-opensource/gennodejs.git
+      version: kinetic-devel
+    release:
+      tags:
+        release: release/lunar/{package}/{version}
+      url: https://github.com/RethinkRobotics-release/gennodejs-release.git
+      version: 2.0.1-0
+    source:
+      test_pull_requests: true
```

Comment:
I haven't seen that you gave permission to the @ros-pull-request-builder yet. So the PR integration might not be complete yet.